internal/cmd/flags.go

```diff
@@ -73,9 +73,9 @@ var (
 	}
 
 	PflagPluginConfig = &pflag.Flag{
-		Name:      "pluginConfig",
+		Name:      "plugin-config",
```

Comment:
Please see `--plugin-config` related comments in #418

Reply:
This fix will be implemented in #418.